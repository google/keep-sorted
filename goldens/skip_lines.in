Skip two lines:
// keep-sorted-test start skip_lines=2
foo
bar
c
b
a
// keep-sorted-test end

Number of skipped lines is greater than block size, so block is ignored:
// keep-sorted-test start skip_lines=10
z
y
x
// keep-sorted-test end

Skipping lines with nested blocks:
<!-- keep-sorted-test start skip_lines=1 -->
This line should not be sorted.
### Table 2
  <!-- keep-sorted-test start skip_lines=2 -->
  Name    | Value
  ------- | -----
  Bravo   | Foxtrot
  Alpha   | Romeo
  <!-- keep-sorted-test end -->
### Table 1
  <!-- keep-sorted-test start skip_lines=2  -->
  Name    | Value
  ------- | -----
  Delta   | Qux
  Bravo   | Bar
  Alpha   | Foo
  Charlie | Baz
  <!-- keep-sorted-test end -->
<!-- keep-sorted-test end -->

Skip lines with group_prefixes:
// keep-sorted-test start skip_lines=2 group_prefixes=:
| birds        |
| ------------ |
| blue footed  |
: booby        |
| crested      |
: bobwhite     |
| anhinga      |
// keep-sorted-test end

Skip lines with group_prefixes indented:
// keep-sorted-test start skip_lines=2 group_prefixes=:
    | birds        |
    | ------------ |
    | blue footed  |
    : booby        |
    | crested      |
    : bobwhite     |
    | anhinga      |
// keep-sorted-test end
