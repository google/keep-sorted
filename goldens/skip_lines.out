Skip two lines:
// keep-sorted-test start skip_lines=2
foo
bar
a
b
c
// keep-sorted-test end

Number of skipped lines is greater than block size, so block is ignored:
// keep-sorted-test start skip_lines=10
z
y
x
// keep-sorted-test end

Skipping lines with nested blocks:
<!-- keep-sorted-test start skip_lines=1 -->
This line should not be sorted.
### Table 1
  <!-- keep-sorted-test start skip_lines=2  -->
  Name    | Value
  ------- | -----
  Alpha   | Foo
  Bravo   | Bar
  Charlie | Baz
  Delta   | Qux
  <!-- keep-sorted-test end -->
### Table 2
  <!-- keep-sorted-test start skip_lines=2 -->
  Name    | Value
  ------- | -----
  Alpha   | Romeo
  Bravo   | Foxtrot
  <!-- keep-sorted-test end -->
<!-- keep-sorted-test end -->
